core/trino-main/src/main/java/io/trino/execution/scheduler/EventDrivenTaskSourceFactory.java

```diff
@@ -49,10 +49,10 @@
 import static io.trino.SystemSessionProperties.getFaultTolerantExecutionTargetTaskInputSize;
```

Comment:
please emphirically test that scaling still works for unpartitioned inserts (in `LocalPlanner` too)